recipes/parsimonious/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "parsimonious" %}
+{% set version = "0.8.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ae0869d72a6e57703f24313a5f5748e73ebff836e6fe8b3ddf34ea0dc00d086b" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python ==2.7|>=3.4
```

Comment:
Please remove the versions for Python here too

Reply:
done. out of curiosity, what happens if someone on python 3.2 (for instance) tries to install the package? It isn't built to support 3.2 --- would installation proceed as normal and the user would just get a runtime error when they hit an incompatibility?